python/lsst/ts/watcher/model.py

```diff
@@ -155,23 +155,37 @@ def __init__(self, domain, config, alarm_callback=None):
 
     @property
     def enabled(self):
-        """Get or set the enabled state of the Watcher model."""
+        """Get the enabled state of the Watcher model."""
         return self._enabled
 
-    def enable(self):
+    async def enable(self):
         """Enable the model. A no-op if already enabled."""
+        self.enable_task = asyncio.create_task(self._enable_impl())
+        await self.enable_task
+
+    async def _enable_impl(self):
+        """Implement of the enable method.
```

Comment:
`Implement` -> `Implementation`? Or `Implement of` -> `Implement`?

Reply:
Ouch. Thank you for catching that!